test/edu-benefits/5495/config/schoolSelection.unit.spec.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import ReactTestUtils from 'react-addons-test-utils';
+
+import { DefinitionTester, submitForm } from '../../../util/schemaform-utils.jsx';
+import formConfig from '../../../../src/js/edu-benefits/5495/config/form.js';
+
+describe('Edu 5495 schoolSelection', () => {
```

Comment:
Is this just to test that it works with the 5495 schema? (What with it being a shared page and all.)